### PR TITLE
v8.11 CHANGELOG.md updates

### PR DESCRIPTION
**Pull request by fallwith**
* Fix bolding of the bugfix
* Standardize formatting across the 2 entries for this release for referencing PRs and community members

### CODE REVIEW

**Review by kaylareopelle — COMMENTED** [review]
These look great! Thanks for the updates. Have you looked to see if we're missing entries for other work that will be included in 8.11? For example, the spelling PR comes to mind.

**Comment by fallwith** [comment]
> These look great! Thanks for the updates. Have you looked to see if we're missing entries for other work that will be included in 8.11? For example, the spelling PR comes to mind.

Thanks, @kaylareopelle. Good call. Addressed with [3728596](https://github.com/newrelic/newrelic-ruby-agent/pull/1530/commits/37285966774da3363203715738f1a95035a34516)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,41 @@
...
+
+
+  * **Cleanup: Performance tests, constants, rubocop-minitest assertions and refutations**
+
+    Community member [@esquith](https://github.com/esquith) contributed a whole slew of cleanup successes for our performance test configuration, orphaned constants in our code base, and RuboCop related improvements. [PR#1406](https://github.com/newrelic/newrelic-ruby-agent/pull/1406) [PR#1408](https://github.com/newrelic/newrelic-ruby-agent/pull/1408) [PR#1409](https://github.com/newrelic/newrelic-ruby-agent/pull/1409) [PR#1411](https://github.com/newrelic/newrelic-ruby-agent/pull/1411)
+
+    Thank you [@esquith](https://github.com/esquith) for these great contributions!
+
+
+  * **CI: nofify on a change from failure to success**
```
Comment (kaylareopelle) [inline CHANGELOG.md:26]:
```suggestion
  * **CI: Nofify on a change from failure to success**
```

**Review by tannalynn — DISMISSED** [review]
LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,41 @@
...
+
+    A super handy, much beloved feature of certain CI and build systems is to not only notify when builds start to fail, but also to notify again when the builds once again start to go green. Community member [@luigieai](https://github.com/luigieai) was able to figure out how configure our existing complex, multiple-3rd-party-action based GitHub Actions pipeline to notify on a switch back to success from failure. [PR#1519](https://github.com/newrelic/newrelic-ruby-agent/pull/1519)
+
+    This is much appreciated! Thank you, [@luigieai](https://github.com/luigieai).
+
+
+  * **Spelling corrections**
+
+    Community member [@jsoref](https://github.com/jsoref), author of the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action, contributed a significant number of spelling corrections throughout the code base. The intelligent issues that were flagged made for a more comprehensive once over than a simple dictionary based check would have been able to provide, and the changs are much appreciated. [PR#1508](https://github.com/newrelic/newrelic-ruby-agent/pull/1508)
```
Comment (hannahramadan) [inline CHANGELOG.md:35]: .... and the `changes` are much appreciated

**Review by hannahramadan — DISMISSED** [review]
Left comments for 2 spelling/grammar edits and then I think looks good!

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,14 +6,41 @@
 
     Previously, the `newrelic deployments` command only supported the older version of the deployments api, which does not currently support newer license keys. Now you can use the New Relic REST API v2 to record deployments by providing your user api key to the agent configuration using `api_key`. When this configuration option is present, the `newrelic deployments` command will automatically use the New Relic REST API v2 deployment endpoint. [PR#1461](https://github.com/newrelic/newrelic-ruby-agent/pull/1461)
 
-    Thank you to @Arkham for bringing this to our attention!
+    Thank you to [@Arkham](https://github.com/Arkham) for bringing this to our attention!
 
 
-  * **Bugfix: Fix memory leak in the Curb instrumentation
+  * **Bugfix: Fix memory leak in the Curb instrumentation**
```
[the remark this answers is not in the document]
Comment (fallwith) [reply CHANGELOG.md]: addressed with [a5775bb](https://github.com/newrelic/newrelic-ruby-agent/pull/1530/commits/a5775bbe0394029d42f5343dcdc6571afee8cedb)

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,14 +6,41 @@
...
+
+    Community member [@esquith](https://github.com/esquith) contributed a whole slew of cleanup successes for our performance test configuration, orphaned constants in our code base, and RuboCop related improvements. [PR#1406](https://github.com/newrelic/newrelic-ruby-agent/pull/1406) [PR#1408](https://github.com/newrelic/newrelic-ruby-agent/pull/1408) [PR#1409](https://github.com/newrelic/newrelic-ruby-agent/pull/1409) [PR#1411](https://github.com/newrelic/newrelic-ruby-agent/pull/1411)
+
+    Thank you [@esquith](https://github.com/esquith) for these great contributions!
+
+
+  * **CI: nofify on a change from failure to success**
+
+    A super handy, much beloved feature of certain CI and build systems is to not only notify when builds start to fail, but also to notify again when the builds once again start to go green. Community member [@luigieai](https://github.com/luigieai) was able to figure out how configure our existing complex, multiple-3rd-party-action based GitHub Actions pipeline to notify on a switch back to success from failure. [PR#1519](https://github.com/newrelic/newrelic-ruby-agent/pull/1519)
```
[the remark this answers is not in the document]
Comment (fallwith) [reply CHANGELOG.md]: addressed with [a5775bb](https://github.com/newrelic/newrelic-ruby-agent/pull/1530/commits/a5775bbe0394029d42f5343dcdc6571afee8cedb) - sorry I misread your commit suggestion as a comment

File: CHANGELOG.md (hunk 2 above)
Comment (fallwith) [reply CHANGELOG.md]: addressed with [24855d4](https://github.com/newrelic/newrelic-ruby-agent/pull/1530/commits/24855d46ce9a7901a55021ea6830b328a476fec6)

**Review by kaylareopelle — APPROVED** [review]
Thanks for accommodating our requested changes!

**Pull request merged**